### PR TITLE
killed the ability to spam emote thumbs down on any message, pep8

### PR DESCRIPTION
**Pull request by aramlaka**
fuck you

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -221,43 +239,49 @@ async def run_cmd(cmd, decode=True):
...
+    'debug': dict(f=handle_debug,   info='sends debug information')
 }
 
+
 @client.event
 async def on_ready():
     print("The bot is ready!")
     await status()
 
+def is_allowed_in_channel(channel):
```
Comment (aramlaka) [inline bot.py:265]: lmaoexdddd

Comment (Chasesc) [reply bot.py:265]: ok

File: bot.py [hunk 2]
```diff
@@ -304,7 +304,11 @@ async def on_reaction_add(reaction, user):
         for auto_added_image_details in queue:
             if reaction.message.id == auto_added_image_details['bot_message_id']:
                 path = Path(
-                    auto_added_image_details['auto_added_image'].decode())
+                    auto_added_image_details
```
Comment (Chasesc) [inline bot.py]: what is wrong with you?

Comment (aramlaka) [reply bot.py]: outdated

File: bot.py [hunk 3]
```diff
@@ -273,21 +297,21 @@ async def on_message(message):
...
 
+
 @client.event
 async def on_reaction_add(reaction, user):
-    if reaction.emoji == '👎':
+    if is_allowed_in_channel(reaction.message.channel) and reaction.emoji == '👎' and reaction.count == 1:
         for auto_added_image_details in queue:
             if reaction.message.id == auto_added_image_details['bot_message_id']:
-                path = Path(auto_added_image_details['auto_added_image'].decode())
+                path = Path(auto_added_image_details['auto_added_image' ].decode())
```
Comment (Chasesc) [inline bot.py]: 😢

**Comment by Chasesc** [comment]
nice

**Pull request merged**